src/components/List/Separator/Separator.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import classnames from './Separators.scss';
+import { bem } from '../../../utils/bem';
+
+export interface Props<> {
+    /** title of the section: optional */
+    title?: string;
+}
+
+const titleBlock = bem('SectionTitle', classnames);
+const noTitleBlock = bem('Divider', classnames);
+
+const Separator = React.forwardRef<HTMLLIElement, Props>(({ title }, ref) => {
+    let block;
+    if (title) {
+        block = titleBlock.block;
+    } else {
+        block = noTitleBlock.block;
+    }
```

Comment:
nice way of managing the classes :-) 

Maybe we can make this code shorter though
```
const block = title ? titleBlock.block : noTitleBlock.block
```